### PR TITLE
[WIP] Adds Travis CI support

### PR DESCRIPTION
**Pull request by luis-pereira**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  script:
+    - locale
+    - cmake -DCMAKE_BUILD_TYPE=Debug -DCMAKE_PREFIX_PATH=/usr -DBUILD_TESTS=YES -DBUILD_DEV_UTILS=YES .
+    - make all
+
+    # run the tests using the xvfb-run wrapper
+    - xvfb-run --auto-servernum ctest --output-on-failure
+
+script:
+  - "curl -s https://raw.githubusercontent.com/mikkeloscar/arch-travis/master/arch-travis.sh | bash"
```
Comment (palinek) [inline .travis.yml]: I'm a bit confused about blindly trusting 3rd party script...

Comment (luis-pereira) [inline .travis.yml]: @palinek Don't worry. This is a WIP.

**Comment by luis-pereira** [comment]
@palinek  A little offtopic: Travis doesn't seem to support dependent build well. That will be a problem for the rest of lxqt

**Comment by agaida** [comment]
@luis - you could try to use my debian snapshot repo - as long there are no major transitions it should work fine

**Comment by luis-pereira** [comment]
@agaida Not sure I understood the meaning of your words. Can you  elaborate ?

**Pull request closed without merging**